_posts/2017-12-12-my-experience-at-js-crash-course.markdown

```diff
@@ -0,0 +1,57 @@
+---
+layout: post
+title: My experience at the JavaScript Crash Course Berlin
+date: 2017-12-12
+categories: story
+---
+
+I recently had the opportunity to attend the [JavaScript Crash Course Berlin](http://wtmberlin.com/javascript-crash-course/).
```

Comment:
can you rename `JavaScript Crash Course Berlin` to `JavaScript Crash Course by Women Techmakers Berlin`?